frame/staking/src/lib.rs

```diff
@@ -328,17 +328,34 @@ use frame_system::{
 };
 use sp_npos_elections::{
 	ExtendedBalance, Assignment, ElectionScore, ElectionResult as PrimitiveElectionResult,
-	to_support_map, EvaluateSupport, seq_phragmen, generate_solution_type, is_score_better,
-	SupportMap, VoteWeight, CompactSolution, PerThing128,
+	to_support_map, EvaluateSupport, seq_phragmen, is_score_better, CompactSolution, SupportMap,
+	VoteWeight, PerThing128,
 };
 pub use weights::WeightInfo;
 
+/// The default solution type used in substrate-node with 24 maximum winners. A runtime can customize this to their needs.
+pub mod default_solution {
```

Comment:
Honestly I put this mostly here so that it can be blindly used in tests/benchmarks etc. Polkadot or any other chain should probably generate their own version.